mlir/include/mlir/Dialect/Rock/Tuning/QuickTuningPerfconfigs.inc

```diff
@@ -0,0 +1,353 @@
+    {16,16,4,16,16,8,1,true,true},
+    {256,128,8,128,32,8,1,true,true},
+    {256,64,2,64,64,8,1,true,true},
+    {64,64,4,32,32,8,1,true,true},
+    {128,128,2,32,32,8,1,true,true},
+    {64,16,8,16,16,16,1,true,true},
+    {128,64,8,32,64,8,1,true,true},
+    {256,256,8,64,32,8,1,true,true},
+    {64,128,8,64,32,8,1,true,true},
+    {128,64,2,64,32,8,1,true,true},
+    {128,256,2,64,32,8,1,true,true},
+    {16,16,8,16,16,8,1,true,true},
+    {128,32,2,32,32,8,1,true,true},
+    {128,256,8,128,32,8,1,true,true},
+    {32,128,2,32,32,8,1,true,true},
+    {64,256,4,32,64,8,1,true,true},
+    {64,32,8,32,32,8,1,true,true},
+    {64,256,2,64,64,8,1,true,true},
+    {16,32,4,16,16,16,1,true,true},
+    {16,32,4,16,32,8,1,true,true},
+    {64,16,8,16,16,8,1,true,true},
+    {256,128,4,32,64,8,1,true,true},
+    {128,256,4,64,32,8,1,true,true},
+    {128,128,4,64,64,8,1,true,true},
+    {16,128,8,16,16,8,1,true,true},
+    {128,16,8,32,16,8,1,true,true}
+};
+// END_CONV_Wmma_f16_DEFS
+
+// BEGIN_GEMM_Wmma_i8_DEFS
+const InitParamsAccel PopulateParamsWmma::initParametersForward8BitGemm[PopulateParamsWmma::nInitParametersForward8BitGemm] = {
+    {128,64,8,32,64,16,1,true,true},
+    {64,128,8,64,32,16,1,true,true},
+    {128,128,4,64,64,16,1,true,true},
+    {64,32,8,16,16,16,1,true,true},
+    {256,32,2,64,32,16,1,true,true},
+    {256,128,4,16,128,16,1,true,true},
+    {256,256,8,128,32,8,1,true,true},
+    {256,128,2,128,32,16,1,true,true},
+    {64,64,4,32,16,16,1,true,true},
+    {16,128,8,16,16,16,1,true,true},
+    {64,256,4,64,64,8,1,true,true},
+    {64,32,4,16,32,16,1,true,true},
+    {16,32,4,16,16,8,1,true,true},
+    {16,16,4,16,16,8,1,true,true},
+    {128,128,4,128,16,16,1,true,true}
+};
+// END_GEMM_Wmma_i8_DEFS
+
+// BEGIN_CONV_Wmma_i8_DEFS
+const InitParamsAccel PopulateParamsWmma::initParametersForward8BitConv[PopulateParamsWmma::nInitParametersForward8BitConv] = {
+    {128,64,8,32,64,16,1,true,true},
+    {32,64,4,32,32,16,1,true,true},
+    {64,64,4,64,16,16,1,true,true},
+    {256,64,8,32,64,16,1,true,true},
+    {128,32,4,64,16,16,1,true,true},
+    {256,32,8,32,16,4,1,true,true},
+    {32,256,4,32,16,4,1,true,true},
+    {128,128,4,128,16,16,1,true,true},
+    {64,16,8,32,16,16,1,true,true},
+    {128,128,2,128,32,16,1,true,true},
+    {256,128,2,32,128,16,1,true,true}
+};
+// END_CONV_Wmma_i8_DEFS
+
+#endif
+
+#ifdef Wmma_DECLARATIONS_GEN
+
+// BEGIN_GEMM_Wmma_f16_DECS
+static constexpr size_t nInitParametersFp16Gemm = 17;
+static const InitParamsAccel initParametersFp16Gemm[nInitParametersFp16Gemm];
+// END_GEMM_Wmma_f16_DECS
+
+// BEGIN_CONV_Wmma_f16_DECS
+static constexpr size_t nInitParametersFp16Conv = 26;
+static const InitParamsAccel initParametersFp16Conv[nInitParametersFp16Conv];
+// END_CONV_Wmma_f16_DECS
+
+// BEGIN_GEMM_Wmma_i8_DECS
+static constexpr size_t nInitParametersForward8BitGemm = 15;
+static const InitParamsAccel initParametersForward8BitGemm[nInitParametersForward8BitGemm];
```

Comment:
is this only for forward? otherwise we can keep names consistent.

Reply:
Forward is relevant for conv, but for gemm it doesn't seem quite right to keep it there. I'll correct that.